sympy/printing/latex.py

```diff
@@ -716,25 +716,35 @@ def _print_Derivative(self, expr):
             diff_symbol = r'\partial'
         else:
             diff_symbol = r'd'
-
         tex = ""
         dim = 0
         for x, num in reversed(expr.variable_count):
-            dim += num
+            dim = dim + num
             if num == 1:
                 tex += r"%s %s" % (diff_symbol, self._print(x))
             else:
                 tex += r"%s %s^{%s}" % (diff_symbol,
                                         self.parenthesize_super(self._print(x)),
                                         self._print(num))
+        neg = False
```

Comment:
This can be handled in a single line when needed below.

Reply:
Ok i will remove variable neg and commit again.